src/app-layout/visual-refresh-toolbar/drawer/global-drawer.tsx

```diff
@@ -133,7 +133,12 @@ function AppLayoutGlobalDrawerImplementation({
                   variant="icon"
                 />
               </div>
-              <div className={styles['drawer-content']}>{activeGlobalDrawer?.content}</div>
+              <div
+                className={styles['drawer-content']}
+                style={{ blockSize: `calc(100vh - ${drawersTopOffset}px - ${placement.insetBlockEnd}px)` }}
```

Comment:
I considered that, or even adding it as another item returned from `getDrawerTopOffset()`. However, when it comes to inline styles, I always find it easier to read when they are truly inline, because they are somewhat of an identifier, like class names/test-ids. Is that just a personal preference?

Reply:
Most likely, all of these lines have to be updated same time if something changed. Reusing the value makes it easier to not forget about it. We have got this misalignment in the past https://github.com/cloudscape-design/components/pull/2744

reusing the constant resolves it

Reply:
Ok. Updated in latest